mkt/settings.py

```diff
@@ -1001,8 +1001,13 @@ def JINJA_CONFIG():
 READ_ONLY = False
 
 REDIS_BACKENDS = {
```

Comment:
For docker, all we need is the host, never seen any of other stuff change. Just setting `REDIS_HOST` as an environment variable would work for me.